### PR TITLE
commands: Add helpful instuctions after "hugo new site"

### LINKED ISSUE #1164: `hugo new site` should provide helpful instructions
After the site skeleton is created it should tell steps on what to do next.

### PR DESCRIPTION
**Pull request by digitalcraftsman**
Fixes #1164

### CODE REVIEW

File: commands/new.go [hunk 1]
```diff
@@ -150,7 +150,13 @@ func doNewSite(basepath string, force bool) error {
 
 	createConfig(basepath, configFormat)
 
-	jww.FEEDBACK.Printf("Congratulations! Your new Hugo site is created in %q.\n", basepath)
+	jww.FEEDBACK.Printf("Congratulations! Your new Hugo site is created in %q.\n\n", basepath)
```
Comment (bep) [inline commands/new.go]: What is wrong with using backtcks to signal a multiline string?

Comment (digitalcraftsman) [inline commands/new.go]: It's definitely the better approach.

**Comment by moorereason** [comment]
lgtm

**Comment by digitalcraftsman** [comment]
Should I merge? It would be part of #2017.

**Comment by bep** [comment]
Yes, please merge.

**Comment by digitalcraftsman** [comment]
Merged as d3e829508b0925c37fdd1548bdebc5fba4e4eb7e.

**Pull request closed without merging**